src/commands/domain/remove.ts

```diff
@@ -0,0 +1,54 @@
+import chalk from "chalk";
+import inquirer from "inquirer";
+
+import { Arguments, createCommand } from "../../util";
+import { requireApp, requireClusterConfig, requireUser } from "../middleware";
+import { cliContainer } from "../../container";
+import { CustomDomain } from "../../container/types";
+
+function confirm(domain: string) {
+  return inquirer.prompt([
+    {
+      message: `Are you sure to remove domain ${domain}?`,
+      name: "proceed",
+      type: "confirm",
+    },
+  ]);
+}
+
+async function run(argv: Arguments) {
+  const domain = argv.domain as string;
+  const answers = await confirm(domain);
+  if (!answers.proceed) {
+    return;
+  }
+  const resp = await cliContainer.getDomains(argv.context.app || "");
+  let customDomain: CustomDomain | undefined;
+  resp.custom_domains.forEach(c => {
```

Comment:
We have several `forEach()` patterns to find an element in an array. We can use `find()` instead.